modules/distr-tracing-tempo-object-storage-setup-aws-sts-install.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+//* observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="distr-tracing-tempo-object-storage-setup-aws-sts-install_{context}"]
```

Comment:
Should the ID for the procedure module more closely match the topic title?  Something along the lines of setting up S3 storage with the Security Token Service?

Reply:
I'm matching the ID to the name of the module file, which has its own convention (so that files for this component are grouped and found in the modules directory.